db/models/user_favorite.js

```diff
@@ -11,8 +11,25 @@ module.exports = {
       }
       client.query(query, (error, results) => {
         release();
-        if (err) {
-          callback(err.stack);
+        if (error) {
+          callback(error.stack);
+        }
+        callback(null, results);
+      });
+    });
+  },
+
+  deleteFavoritePattern(userId, patternId, callback) {
+    const query = `DELETE FROM public.user_favorite
+    WHERE pattern_id=${patternId} AND user_id=${userId};`;
```

Comment:
https://node-postgres.com/features/queries

> If you are passing parameters to your queries you will want to avoid string concatenating parameters into the query text directly. This can (and often does) lead to sql injection vulnerabilities. node-postgres supports parameterized queries, passing your query text unaltered as well as your parameters to the PostgreSQL server where the parameters are safely substituted into the query with battle-tested parameter substitution code within the server itself.

```suggestion
    const query = {
      text: 'DELETE FROM public.user_favorite WHERE pattern_id=$1 AND user_id=$2;',
      vaues: [patternId, userId],
    };
```